### DISCUSSION TITLE
How to get the new UI

### DISCUSSION THREAD

**Discussion by steelx** [darkreader/darkreader#6707] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/3184210/132450654-db9c25b1-4732-4e75-99f1-a52da82360e3.png)

I previously had this version of dark reader, but after update I cant see it anymore. As well I installed code from git `https://github.com/darkreader/darkreader/tree/Color-Patch` but still same.

Im looking for the version of DarkReader where you were able to customize the auto colors, even the background shades in app.

**Comment by Gusted** [comment, marked as answer]
Try out, Dev tools -> Preview new design.

**Reply by steelx** [reply]
wow thanks :)

**Comment by InsanityOnABun** [comment]
This now shows the "new" new design, which personally I find awful. Is there a way to get back to the "old" new design?

Ugly "new" new design:
<img width="286" height="401" alt="image" src="https://github.com/user-attachments/assets/27d6ef0b-87ae-492f-bc5f-0cec27b035c8" />

**Reply by Myshor** [reply]
Just to confirm... you're using Edge and downloaded addon from Edge addons page?

**Reply by alexanderby** [reply]
Hi @InsanityOnABun. What do you find awful?

**Reply by InsanityOnABun** [reply]
@Myshor correct, Edge and the Edge Add-Ons Store.

@alexanderby it just feels ugly and tacky. Compared to the "old" new design shown below, it feels unorganized, tacky with colors, and upsalesy with the big "UPGRADE" button and "Premium themes". Multiple themes are also something I do not care in the slightest about and just add to visual clutter personally.

I did ultimately find a way to get back to the old new layout — using Inspector on the settings pop-out window, setting up a local override of the JS file that houses the functions called when the new / old layout buttons are pressed, and changing the method called to the old one that is still in there, but normally not called anywhere.

<img width="286" height="532" alt="image" src="https://github.com/user-attachments/assets/bb549c05-0771-4758-93b2-71eb6b383db3" />

**Reply by Myshor** [reply]
@InsanityOnABun for today you can try alternative version from Chrome Web Store. As I remember that new layout is now pushed to Edge store only.

I can not promise that next updates won't change layout even in Chrome Web Store versions but for today it's alternative. 😉

**Reply by chatmandu-uk** [reply]
I prefer the new layout and I probably wouldn't have found the theme colour options without seeing it in MS Edge.

**Comment by mpaunovic** [comment]
Just a quick thank you for the new UI redesign, I really love the new minimal look.

It feels clean, focused, and much more streamlined without losing any functionality. The simplicity makes the whole experience smoother and more enjoyable.

Thanks for the great work and for continuing to improve such a fantastic extension!
